### PR TITLE
2024 announce

### PR DESCRIPTION
**Pull request by garrying**

### CODE REVIEW

File: _layouts/splash.html [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+  <header>
+    <h1 class="font-display text-sm">Our Networks 2024</h1>
+    <h2 class="font-display text-4xl mt-0">Does not (cloud) compute</h2>
+    <h3 class="font-display text-2xl">July 27, 2024</h3>
+    <h4 class="font-display text-2xl mb-0">
+      <a href="https://312main.ca/" target="_blank">312 Main</a>
+    </h4>
+    <h5 class="font-display text-2xl mt-0 mb-0">Vancouver, BC</h5>
   </header>
-  {{ content }}
```
Comment (dcwalk) [inline _layouts/splash.html:8]: we talked about updating this ascii image and changing what is in the laptop screen

File: _includes/footer.html [hunk 2]
```diff
@@ -10,3 +10,17 @@
     </div>
   </div>
 </footer>
```
Comment (dcwalk) [inline _includes/footer.html:11]: the more I think about this footer image I think that some other jgs might feel personally more suited to the location and my interpretation of look and feel. I'll add to comment at end of review

**Review by dcwalk — COMMENTED** [review]
From our review yesterday we identified this as needed before merging:

- add credit for ascii art and font to about page
- integrate the divider style more to other content / pages

After looking at other jgs ascii art I feel relatively strong that we can find one of their works that feels more suited to time and place (Vancouver where sea meets the sky) for the footer.

If what was interesting was how they represented organic things with ascii symbols, organic things that reflect the actual landscape of vancouver are: trees, oceans, mountains. 

Could we try the footer with ascii art that references those?

Some jgs options:
```
                         |    
                     \       /            _\/_
                       .-'-.              //o\  _\/_
    _  ___  __  _ --_ /     \ _--_ __  __ _ | __/o\\ _
  =-=-_=-=-_=-=_=-_= -=======- = =-=_=-=_,-'|"'""-|-,_ 
   =- _=-=-_=- _=-= _--=====- _=-=_-_,-"          |
  jgs=- =- =-= =- = -  -===- -= - ."
  
        _    .  ,   .           .
    *  / \_ *  / \_      _  *        *   /\'__        *
      /    \  /    \,   ((        .    _/  /  \  *'.
 .   /\/\  /\/ :' __ \_  `          _^/  ^/    `--.
    /    \/  \  _/  \-'\      *    /.' ^_   \_   .'\  *
  /\  .-   `. \/     \ /==~=-=~=-=-;.  _/ \ -. `_/   \
 /  `-.__ ^   / .-'.--\ =-=~_=-=~=^/  _ `--./ .-'  `-
/jgs     `.  / /       `.~-^=-=~=^=.-'      '-._ `._

    .                  .-.    .  _   *     _   .
           *          /   \     ((       _/ \       *    .
         _    .   .--'\/\_ \     `      /    \  *    ___
     *  / \_    _/ ^      \/\'__        /\/\  /\  __/   \ *
       /    \  /    .'   _/  /  \  *' /    \/  \/ .`'\_/\   .
  .   /\/\  /\/ :' __  ^/  ^/    `--./.'  ^  `-.\ _    _:\ _
     /    \/  \  _/  \-' __/.' ^ _   \_   .'\   _/ \ .  __/ \
   /\  .-   `. \/     \ / -.   _/ \ -. `_/   \ /    `._/  ^  \
  /  `-.__ ^   / .-'.--'    . /    `--./ .-'  `-.  `-. `.  -  `.
@/        `.  / /      `-.   /  .-'   / .   .'   \    \  \  .-  \%
@&8jgs@@%% @)&@&(88&@.-_=_-=_-=_-=_-=_.8@% &@&&8(8%@%8)(8@%8 8%@)%
@88:::&(&8&&8:::::%&`.~-_~~-~~_~-~_~-~~=.'@(&%::::%@8&8)::&#@8::::
`::::::8%@@%:::::@%&8:`.=~~-.~~-.~~=..~'8::::::::&@8:::::&8:::::'
 `::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::.'
 ```

and another one:
```
                                /^L_      ,."\
           /~\       __       /~    \   ./    \
          /   _\   _/  \     /T~\|~\_\ / \_  /~|          _^
        / \ /W  \ / V^\/X  /~         T  . \/   \    ,v-./
 ,'`-. /~   ^     H  ,  . \/    ;   .   \      `. \-'   /
     M      ~     | . ;  /         ,  _   :  .    ~\_,-'
    /    ~    .    \    /   :                   '   \   ,/`
   I o. ^    oP     '98b         -      _  9.`       `\9b.
 8oO888.  oO888P  d888b9bo. .8o 888o.       8bo.  o     988o.
 88888888888888888888888888bo.98888888bo.    98888bo. .d888P
 88888888888888888888888888888888888888888888888888888888888
 88888888888888P"   "" "   """9888P" P" "8P"   ""*9888888888
```
by Veer Kothari + Ojosh!ro

File: _includes/footer.html (hunk 2 above)
Comment (garrying) [reply _includes/footer.html:11]: I went with the second jgs piece your comment.

File: _layouts/splash.html (hunk 1 above)
Comment (garrying) [reply _layouts/splash.html:8]: Updated with rotating slogans

**Comment by dcwalk** [comment]
@garrying  apologies on my slow reply -- looks amazing!

However, the two ascii works now have scroll bats for me, I tested on both chromium and firefox with same behaviour.  Also the footer feels like it's missing maybe a bit of spacing before the end of the page (unless that is intentional) . screenshot attached

image

**Pull request merged**